src/app/shared/components/filters-list/categories-filter/categories-filter.component.html

```diff
@@ -6,9 +6,8 @@
     <div class="category_details">
       <h3 class="category_title">Спорт</h3>
       <div class="category_link">
-        <a href="#"> 254 гуртки <span class="material-icons arrow">
-        arrow_right_alt
-        </span>
+        <a routerLink="/result" (click)="selectCategory('sport')"> 
```

Comment:
since we need to have possibility to change category name it would be better not to rely on the name but on id